raccoongang/edx-platform
# added CMS configuration variables for 'storages.backends.azure_storage.AzureStorage'

**Description:** Added Azure storage credentials variables into CMS configuration.

**Youtrack:** https://youtrack.raccoongang.com/issue/CSALS-29

**Testing instructions:**

1. Install SCROM xblock onto platform (https://github.com/raccoongang/edx_xblock_scorm);
2. Configure CMS to use Azure storage:
```
"DEFAULT_FILE_STORAGE": "storages.backends.azure_storage.AzureStorage",
"AZURE_ACCOUNT_KEY": "azure_key",
"AZURE_ACCOUNT_NAME": "storage_name"
```

3. Try to upload scorm file in scrorm unit properties;
3. Expect successful file upload;
4. If error displayed - check failed.
